robocop/checkers/spacing.py

```diff
@@ -38,8 +43,12 @@ def __init__(self, *args):
     def parse_file(self):
         self.lines = []
         super().parse_file()
-        if self.lines and not self.lines[-1].endswith('\n'):
-            self.report("missing-trailing-blank-line", lineno=len(self.lines), col=0)
+        if self.lines:
+            last_line = self.lines[-1]
+            if not last_line.endswith('\n'):
+                self.report("missing-trailing-blank-line", lineno=len(self.lines), col=0)
+            if last_line == '\n':
```

Comment:
what happens for this case (extra spaces before new line char):
```
    Keyword\n
    \n
```

Reply:
Probably it fails :) Good catch. I will cover this case.

Reply:
Just in case - you can add several files to single atest (for example file with trailing line, file without, file with spaces and \n..). expected_output will then contain merged issues from all files.